.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: c
+sudo: required
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      sudo: required
+    - os: linux
+      dist: precise
+      sudo: required
+install:
+- sudo apt-get update
+- sudo apt-get install --yes build-essential autoconf libtool gawk alien fakeroot gdebi linux-headers-generic
+- sudo apt-get install zlib1g-dev uuid-dev libattr1-dev libblkid-dev libselinux-dev libudev-dev parted lsscsi wget ksh
+script:
+- export MAKEFLAGS="-j1"; echo $MAKEFLAGS
```

Comment:
@behlendorf `MAKEFLAGS="-jN"` with N>1 seems to be (IMHO) the root cause of https://github.com/zfsonlinux/zfs/issues/2948. We were only trying to confirm that.